encord/objects/ontology_object_instance.py

```diff
@@ -816,13 +874,18 @@ def from_dict(d: dict) -> ObjectInstance.FrameInfo:
             else:
                 last_edited_at = datetime.now()
 
+            if "createdAt" in d and d["createdAt"] is not None:
+                created_at = parse_datetime(d["createdAt"])
+            else:
+                created_at = datetime.now()
```

Comment:
😮 
This doesn't make sense to me.
On creation yes, we should be providing a created_at timestamp. But I strongly disagree that on fail find, we should fallback to providing now as the created_at. I'd rather have None or the Unix Epoch time than now.

If you are using this branch as a mechanism somehow of updating created_at, if this is the first time creating labels, then I would reconsider refactoring such that it doesn't have this risky side effect.

Reply:
Ah that is very correct! I put this in at the start just to get it working. CreatedAt should always be there, no defaults required.